Features/Shop.feature

```diff
@@ -1,8 +1,118 @@
-﻿Feature: Shop Page
+﻿@PageSetup
+
+Feature: Shop
+
+As a User, I want to group products into tabs,
+I can filter products by values on those tabs,
+I can add items to the cart, 
+and remove added items from the cart.
+
+Scenario: Verify every Tab <h1>:
+
+	#Arrange:
+	Given I am on 'shop' page
+	#Act:
+	When I click on '<Tab Name>' tab
+	#Assert:
+	Then I see '<h1>' heading is displayed
+
+Examples:
+	| Tab Name           | h1                |
+	| Сонячні панелі     | Сонячні панелі    |
+	| Інвертори          | Сонячні інвертори |
+	| Акумулятори        | Акумулятори       |
+	| Контролери заряду  | Контролери заряду |
+	| Системи кріплення  | Системи кріплення |
+	| Кабель і комутація | Сонячний кабель   |
+
+
+#Verify that catalog filter works correctly
+
+Scenario: Verify Filter on the tab:
+
+	#Arrange:
+	Given I am on 'shop/<Tab>' page
+	#Act:
+	When I click on Filter button
+	And I click on '<FilterValue>' checkbox
+	#Assert:
+	Then I see '<FilterValue>' filtered products
+
+Examples:
+	| Tab          | FilterValue    |
+	| inverters    | Huawei         |
+	| inverters    | Deye           |
+	| inverters    | Fronius        |
+	| inverters    | LuxPower       |
+	| inverters    | Victron Energy |
+	| solar-panels | SOLA           |
+	| solar-panels | Abi-Solar      |
+	| solar-panels | Ulica Solar    |
+	| solar-panels | Yingli Solar   |
+	| solar-panels | JA Solar       |
+	| solar-panels | Jinko Solar    |
+	| batteries    | AXIOMA         |
+	| batteries    | BYD            |
+	| batteries    | PYLONTECH      |
+	#| solar-panels | C&T Solar                      | not all contain full Brand name
```

Comment:
this confirms that strategy how to verify filtering is not correct. If your test can filter only by brand (moreover only for some brands) then it means that you verify wrong data.